### PR TITLE
add challenges and beers-app

### PR DESCRIPTION
**Pull request by NuriaComa**

### CODE REVIEW

File: staff/NuriaComa/lottery/scripts/logic.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/**
```
Comment (agandia9) [inline staff/NuriaComa/lottery/scripts/logic.js:1]: use strict?

Comment (NuriaComa) [reply staff/NuriaComa/lottery/scripts/logic.js:1]: shit!

Comment (NuriaComa) [reply staff/NuriaComa/lottery/scripts/logic.js:1]: line 5....

File: staff/NuriaComa/task-app/scripts/main.M.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
+'use strict';
```
Comment (manuelbarzi) [inline staff/NuriaComa/task-app/scripts/main.M.js]: this file seems to be unnecessary, unused. please, remove it. let's keep code clean.

Comment (NuriaComa) [reply staff/NuriaComa/task-app/scripts/main.M.js]: Done!

**Review by agandia9 — COMMENTED** [review]
OK!

**Pull request merged**